FordLabs/PeopleMover
# Add created by field to spaces

## Issue
Connects #367

## What was done
- [x] Add created by field to spaces

## How to test
1. Create a space and then ensure that a createdBy property was added with your cdsId
